src/builder-flatpak-utils.c

```diff
@@ -67,10 +67,13 @@ flatpak_write_update_checksum (GOutputStream  *out,
                                gconstpointer   data,
                                gsize           len,
                                gsize          *out_bytes_written,
-                               GChecksum      *checksum,
+                               GChecksum     **checksums,
+                               int             n_checksums,
```

Comment:
`gsize` for array lengths, not `int`.

Reply:
Well, its at most 4, but i guess it looks nicer...

Reply:
It also won’t give you problems with signed vs unsigned. I think it’s just a good habit to get into, rather than having to think about the maximum possible length of each array you come across.